questions/qa-scripts.en.html

```diff
@@ -1339,25 +1339,25 @@ <h2>Further reading</h2>
 <li>
 <p>Related links, <cite>Authoring HTML &amp; CSS</cite></p>
 <ul>
-<li><a href="https://www.w3.org/International/techniques/authoring-html#direction">Text direction</a></li>
+<li><a href="/International/techniques/authoring-html#direction">Text direction</a></li>
 </ul>
 </li>
 <li>
-<p><a href="https://www.ethnologue.com/">SIL Ethnologue</a> (A good
-resource for information about languages.)</p>
+<p><a href="https://www.ethnologue.com/">SIL Ethnologue</a>: A good
```

Comment:
I don't mind changing away from the parentheses, but a colon is commonly used to separate main and sub titles. Even though the link is only applied to the 'SIL Ethnologue' text, it still looks a little like the text after the : is a subtitle.  We could perhaps use a full stop rather than a colon (and begin a new sentence with a capital letter).  What do you think?